sdks/java/container/build.gradle

```diff
@@ -68,6 +73,28 @@ golang {
   }
 }
 
+// this is a workaround to call generateLicenseReport task from project root directory.
+// generateLicenseReport does not return correct dependency list when not called from the root.
+task generateThirdPartyLicenses(type: Exec) {
+  workingDir project.rootProject.projectDir
+  commandLine './sdks/java/container/license_scripts/license_script.sh'
```

Comment:
Why does the gradle task need to start a shell script that runs gradle? Can the gradle task be a dependency instead?

Reply:
When the `generateLicenseReport` task is triggered by `dependsOn`, the task is not running at the project root directory regardless how to set project scope, and dependency list is not correct. This is a workaround to execute the `generateLicenseReport` from root directory and get correct dependency list.